liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -253,8 +313,19 @@ public class DevMojo extends LooseAppSupport {
     /**
      * If true, preserve the temporary Containerfile/Dockerfile used in the container build command
      */
-    @Parameter(alias="keepTempContainerfile", property = "keepTempContainerfile", defaultValue = "false")
+    private boolean keepTempContainerfile;
     private boolean keepTempDockerfile;
+
+    @Parameter(alias="keepTempContainerfile", property = "keepTempContainerfile", defaultValue = "false")
+    public void setKeepTempContainerfile(boolean keepTempContainerfile) {
+        this.keepTempContainerfile = keepTempContainerfile;
+        this.keepTempDockerfile = keepTempContainerfile;
+    }
+
+    @Parameter(alias="keepTempDockerfile", property = "keepTempDockerfile")
```

Comment:
Yeah, I guess it really just matters if we want to have that precedence when both are set. My thought was that it doesn't really affect anything running so it would be fine to behave that way. If we want to have the precedence, I can change it to use `Boolean` instead.